docker_explorer/de.py

```diff
@@ -164,13 +166,48 @@ def ParseArguments(self):
     return opts
 
   def ParseOptions(self, options):
-    """Parses the command line options.
+    """Parses the command line options."""
+    self.docker_directory = os.path.abspath(options.docker_directory)
 
+  def GetContainer(self, container_id):
+    """Returns a Container object given a container_id.
+
+    Args:
+      container_id (str): the ID of the container.
     Returns:
-      Namespace: the populated namespace.
+      Container: the container's info.
```

Comment:
What is returned is a container.Container object right?

Reply:
done